confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/read/GetRegionInfoListPlan.java

```diff
@@ -49,14 +53,34 @@ public void setRegionType(TConsensusGroupType regionType) {
     this.regionType = regionType;
   }
 
+  public List<String> getStorageGroups() {
+    return storageGroups;
+  }
+
+  public void setStorageGroups(List<String> storageGroups) {
+    this.storageGroups = storageGroups;
+  }
+
+  public boolean isFilterByStorageGroup() {
+    return filterByStorageGroup;
+  }
+
+  public void setFilterByStorageGroup(boolean filterByStorageGroup) {
+    this.filterByStorageGroup = filterByStorageGroup;
+  }
```

Comment:
We can drop these setters if we take the TShowRegionReq as a construct parameter.